dockerfiles/alpine/Dockerfile

```
@@ -1,8 +1,10 @@
 FROM golang:1 as builder
 COPY . /src
 WORKDIR /src
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
```

```suggestion
WORKDIR /src
COPY go.mod .
COPY go.sum .
RUN go mod download
COPY . .
```
I think we might need to move these lines above the `COPY` step so that it is cached desirably. Similar to [Concourse Dockerfile](https://github.com/concourse/concourse/blob/master/Dockerfile#L7-L15)